package-builder/build-manifest.nix

```diff
@@ -1,7 +1,8 @@
 {
-  pkgs ?
-    import (builtins.getFlake "github:flox/nixpkgs/dc14ed91132ee3a26255d01d8fd0c1f5bff27b2f")
-      { },
+  # nixpkgs providing `lib` and `stdenv` (`runCommand`)
+  # this is overridden to point to the nixpkgs used to build flox by the caller
+  nixpkgs-url ? "github:flox/nixpkgs/stable",
```

Comment:
(non-blocking, curious) Does this need to have a default? Will it increase the risk of us misconfiguring it in the future?

Reply:
Speaking for myself, the default is v. useful as I will first run flox with the `-v` flag to discern the full `nix build` command being invoked, then cut & paste that replacing the path to `build-manifest.nix` with a local path for rapid iteration.